WcaOnRails/app/views/incidents/index.html.erb

```diff
@@ -0,0 +1,108 @@
+<% provide(:title, "Incidents log") %>
+
+<%= render layout: "nav" do %>
+  <h1><%= yield(:title) %></h1>
+
+  <div id="custom-search-toolbar">
+    <input id="incident-tags" class="form-control" type="text" placeholder="Filter by tags">
+  </div>
+  <div class="incidents-log-table-container">
+    <%= wca_table(table_class: "incidents-log-table", data: { toggle: "table", pagination: true, search: true, toolbar: "#custom-search-toolbar" }, floatThead: false, greedy: false) do %>
```

Comment:
This does the filtering on the client side, which is certainly fine for now, but if the number of incidents grows, we should do the filtering & pagination on the server side.